lib/index.d.ts

```diff
@@ -0,0 +1,43 @@
+import {Request} from 'express';
+import * as passport from 'passport';
+
+export interface StrategyOptions {
+  clientID: string;
+  clientSecret: string;
+  callbackURL: string;
+  passReqToCallback?: true;
```

Comment:
The `?` means that this is optional and doesn't have to be provided. Since it doesn't make sense to provide it with a `false` option, I've set the type to `true` meaning if it's provided it must be `true`, otherwise it'll be the boolean default when interpreted internally by this module.